Product/Production/Adapters/General/CONNECTConsumerPreferencesProfileGUI/src/main/java/gov/hhs/fha/nhinc/adapter/cppgui/valueobject/PatientVO.java

```diff
@@ -26,12 +26,18 @@
  */
 package gov.hhs.fha.nhinc.adapter.cppgui.valueobject;
 
+
+import java.io.Serializable;
+
+import gov.hhs.fha.nhinc.adapter.cppgui.valueobject.PatientPreferencesVO;
 /**
  *
  * @author patlollav
  */
-public class PatientVO {
-    private String firstName;
+public class PatientVO implements Serializable{
+
+	private static final long serialVersionUID = 1L;
```

Comment:
You created a unique uid in one of the classes below, but used 1L here; any reason for the mix and match?